content/learn/book/getting-started/ecs/_index.md

```diff
@@ -144,3 +144,33 @@ hello Zayna Nieves!
 Marvelous!
 
 **Quick Note**: "hello world!" might show up in a different order than it does above. This is because systems run in parallel by default whenever possible.
+
+If we want to change the names of some people (perhaps they got married!), for example, we can do this using a mutable query:
```

Comment:
Could be useful to add a title or [anchor/id](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_element_on_the_same_page) so readers can be referred to this section in particular ; but I wouldn't block on it.

Reply:
like this?